boards/arm/frdm_k82f/frdm_k82f.dts

```diff
@@ -224,3 +224,13 @@
 &pit0 {
 	status = "okay";
 };
+
+arduino_i2c: &i2c0 {
+	status = "okay";
+	pinctrl-0 = <&I2C0_SDA_PTB3 &I2C0_SCL_PTB2>;
+};
+
```

Comment:
Missing the I2C flags here:

Need something like:

&I2C0_SCL_PTB2 {
   drive-open-drain;
};

&I2C0_SDA_PTB3 {
   drive-open-drain;
};